config/default.yml

```diff
@@ -271,6 +271,7 @@ Performance/Sum:
   Reference: 'https://blog.bigbinary.com/2016/11/02/ruby-2-4-introduces-enumerable-sum.html'
   Enabled: 'pending'
   VersionAdded: '1.8'
+  VersionChanged: '1.9'
```

Comment:
The next release is planned for 1.8.1 and I think this change can be included in the 1.8 series. Could you remove this one?

Reply:
Sure, removed!